src/common/HTTPRequest.ts

```diff
@@ -124,6 +124,16 @@ export class HTTPRequest {
   private _postData?: string;
   private _headers: Record<string, string> = {};
   private _frame: Frame;
+  private _continueRequestOverrides: ContinueRequestOverrides;
+  private _responseForRequest: Partial<ResponseForRequest>;
+  private _abortErrorReason: Protocol.Network.ErrorReason;
+  private _continueRequested: boolean;
```

Comment:
WDYT about changing this to _currentStrategy and _currentPriority instead?

Reply:
Sounds okay, I'll try it and see how it goes.